docs/topics/cache.txt

```diff
@@ -401,6 +403,19 @@ behavior. These arguments are provided as additional keys in the
     On some backends (``database`` in particular) this makes culling *much*
     faster at the expense of more cache misses.
 
+  .. versionchanged:: 1.11
```

Comment:
Ah sorry I missed that part of the guide. So like this?
https://emorley.pastebin.mozilla.org/8905150